### PR TITLE
Update FAQ.rst

### PR DESCRIPTION
**Pull request by SriharshaSubbakrishna**
Corrected some typos and added a question to address Defect and feature request raising.

### CODE REVIEW

**Comment by pankajgoyal2** [comment]
DCO and Jenkins job is failing. You need to add Sign-Off to the commit message.

File: docs/FAQ.rst [hunk 1]
```diff
@@ -90,6 +90,11 @@ File a Pull Request (PR) against this file in the
 Avalon source code. See
 https://github.com/hyperledger/avalon/blob/master/CONTRIBUTING.md
 
+How do I rasie a defect or a new feature request to the hyperledger Avalon community?
+-------------------------------------------------------------------------------------
+You can log any new feature request OR issues/defects you find with the Avalon code, here.
+https://jira.hyperledger.org/secure/RapidBoard.jspa?rapidView=241&projectKey=AV&view=planning.nodetail
```
Comment (Ram-srini) [inline docs/FAQ.rst]: We discussed and decided to use github issues for now. May be you can update the document.

Comment (manojgop) [reply docs/FAQ.rst]: rasie -> raise

**Comment by danintel** [comment]
Merged with PR #315

**Pull request closed without merging**